src/styles/index.scss

```diff
@@ -1,3 +1,134 @@
 body {
   margin: 0;
 }
+
+.product {
+  &-card {
+    padding-left: 16px;
+    display: flex;
+    box-sizing: border-box;
+    width: 200px;
+    flex-direction: column;
+    text-align: center;
+    font-family: Roboto, sans-serif;
```

Comment:
Consider specifying the font family for the entire body to get this style inherited by elements